docs/docs/reference/project-files/apis.md

```diff
@@ -11,8 +11,13 @@ See comprehensive documentation on how to define and use [custom APIs](/integrat
 
 _**`type`**_ — Refers to the resource type and must be `api` _(required)_.
 
+_**`name`**_ — Refers to the name of the API. _(optional)_.
```

Comment:
The `name:` property is not specific to API YAML, it works for all YAML resources. It defaults to the filename without file extension, which is sufficient for basically all use cases – so should we maybe not document it?

Reply:
true, will remove this one.